cmd/soroban-cli/src/commands/contract/init.rs

```diff
@@ -47,6 +47,9 @@ pub struct Cmd {
         long_help = "An optional flag to pass in a url for a frontend template repository."
     )]
     pub frontend_template: String,
+
+    #[arg(short, long, long_help = "Overwrite all existing files.")]
```

Comment:
Is the intent that the command will overwrite the entire directory such as remove the directory contents first, then start writing? Or is the intent that the command will overwrite individual files and leave any additional files as they are?

The term 'all' may introduce some ambiguity to the scope of what is removed, and given the destructive nature of this command we should try and limit the ambiguity.

The term 'any' may limit the ambiguity, although not perfectly.

```suggestion
    #[arg(short, long, long_help = "Overwrite any existing files.")]
```